dask_planner/src/parser.rs

```diff
@@ -255,37 +294,43 @@ impl<'a> DaskParser<'a> {
 
     /// Parse a SQL CREATE statement
     pub fn parse_create(&mut self) -> Result<DaskStatement, ParserError> {
-        let if_not_exists =
-            self.parser
-                .parse_keywords(&[Keyword::IF, Keyword::NOT, Keyword::EXISTS]);
         let or_replace = self.parser.parse_keywords(&[Keyword::OR, Keyword::REPLACE]);
         match self.parser.peek_token() {
             Token::Word(w) => {
                 match w.value.to_lowercase().as_str() {
                     "model" => {
                         // move one token forward
                         self.parser.next_token();
+
+                        let if_not_exists = self.parser.parse_keywords(&[
+                            Keyword::IF,
+                            Keyword::NOT,
+                            Keyword::EXISTS,
+                        ]);
+
                         // use custom parsing
                         self.parse_create_model(if_not_exists, or_replace)
                     }
+                    "schema" => {
+                        // move one token forward
+                        self.parser.next_token();
+
+                        let if_not_exists = self.parser.parse_keywords(&[
+                            Keyword::IF,
+                            Keyword::NOT,
+                            Keyword::EXISTS,
+                        ]);
+
+                        // use custom parsing
+                        self.parse_create_schema(if_not_exists, or_replace)
+                    }
                     "table" => {
                         // move one token forward
                         self.parser.next_token();
                         // use custom parsing
                         self.parse_create_table(or_replace)
```

Comment:
For sake of consistency, think we could modify `parse_create_table` to accept `if_not_exists` so this could match the other cases?

```suggestion
                        self.parser.next_token();

                        let if_not_exists = self.parser.parse_keywords(&[
                            Keyword::IF,
                            Keyword::NOT,
                            Keyword::EXISTS,
                        ]);

                        // use custom parsing
                        self.parse_create_table(if_not_exists, or_replace)
```